llvm/test/Transforms/LoopVectorize/RISCV/conditional-scalar-assignment.ll

```diff
@@ -0,0 +1,2932 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
Can the general code-gen parts be tested independent of RISCV? Could you add a test checking the printed VPlan containing the CAS recipes?

Reply:
There is no way to make a generic target support predicated vectorization to my knowledge. As a result, I have moved these tests back RISC-V. I think we can consider moving them back out until a generic target can handle it. We do something similar for other code that uses this option. WDYT?

Reply:
I think I am missing something, but the non-EVL codegen doens't seem to use and predicated memory accesses, so it should be able to do it with a generic target?